tests/ui/drop/issue-48962.rs

```diff
@@ -10,6 +10,7 @@ impl Drop for Dropee {
     fn drop(&mut self) {
         unsafe {
             ORDER[INDEX] = self.0;
+            //~^ WARN creating a reference to mutable static is discouraged [static_mut_refs]
```

Comment:
Since this test is unrelated, let's fix (or `allow(..)` and mark `FIXME`) this and other similar uses instead.